tests/functional/test_metamodel/test_model_kwargs.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import unicode_literals
+from textx import (metamodel_from_str)
+import os.path
+from pytest import raises
+from textx.exceptions import TextXError
+
+
+grammar = r"""
+Model: 'MyModel' name=ID;
+"""
+
+text = r"""
+MyModel test1
+"""
+
+
+def test_model_kwargs():
+    mm = metamodel_from_str(grammar)
+    mm._tx_model_kwarg_definitions.add_definition(
```

Comment:
This would thus be `_tx_model_param_defs`  and the `add_definition` can just be `add` it is implied what we are adding by the name of the container.

Reply:
done.